DECISIONS.md

```diff
@@ -4,44 +4,51 @@
-    1. Make a set-up/install script so the user does not need to worry about the installation of `node`, `npm`, and packages.
-    1. User Docker containers to manage the environment.
-        - Pro: this would make the environment easy and consistent across all users. Users would only need to worry about a Docker installation.
-            - Good option down the line.
-        - Con: this is a current barrier to devs (learning and time to commit to this project)--but otherwise surmountable on the learning front.
-- HTML to image options: this has to be lightweight; many options open a headless browser reliant on a browser driver (e.g. selenium uses the Chrome driver) which takes a few seconds per conversion. (see posted issue for follow-up on this separately).
+    - Con: may make goal 1 of ease of installation still difficult. We will perform early user testing to see how much of a barrier this is--noting this is an early version of the tool.
+    - Add-on options to consider:
+        1. Include any node.js library binaries in the Python library to negate a `npm` and `node.js` environment for users (assumption: ease of installation comes from Python installation being the only dependency). The difficulty with this route is that binaries for multiple OSs should be included.
+            - **We want to attempt this route first to abstract away JS ecosystem.**
+        1. Make a set-up/install script so the user does not need to worry about the installation of `node`, `npm`, and packages.
+1. Containerize `py-allotax` to manage the environment.
+    - Pro: this would make the environment easy and consistent across all users. Users would only need to worry about a Docker installation.
+    - Con: user testing needed for accessibility goal; also unclear how much maintainability would be affected while the py-allotax's `allotaxonometer` library remains a branch of the main library.
 
+ ### HTML to image options
+
+ This has to be lightweight; many options open a headless browser reliant on a browser driver (e.g. selenium uses the Chrome driver) which takes a few seconds per conversion (see posted issue for follow-up on this separately). We explored numerous options that do not seem possible, so this area remains a task anyone is free to take on. Specifically, we tried `pandoc`, `weasyprint`, node.js `html-to-pdf`, and a less restrictive DOM `linkedom`.
+ - `pandoc`: would have been a good option but had many texlive (latex universe) and system dependencies that ultimately did not work out.
+ - `weasyprint`: we think it could work but may require some tinkering with the CSS elements in the HTML plot template. Library logging showed weasyprint fails to render SVG image and an option wasn't apparent.
+ - `linkedom`: As with the current DOM (`jsdom`), a richer browser was required (headless browser to render styles and flexboxes), making `html-to-pdf` a non-option.
 
 ## Future improvements (TODOs):
```

Comment:
We might just want to point to a file, github project, or issue. If no, I fear that this list will be outdated faster than you can say... Jack Robinson ? Is that a real idiom.